pkg/kubelet/dockertools/docker.go

```diff
@@ -325,7 +326,7 @@ func NewDockerContainerCommandRunner(client DockerInterface) ContainerCommandRun
 }
 
 func (p dockerPuller) Pull(image string) error {
-	image, tag := parseImageName(image)
+	_, tag := parsers.ParseRepositoryTag(image)
```

Comment:
Shadowing 7, Kubernetes team 0


> On Mar 17, 2015, at 6:44 PM, Victor Marmol <notifications@github.com> wrote:
> 
> In pkg/kubelet/dockertools/docker.go:
> 
> > @@ -325,7 +326,7 @@ func NewDockerContainerCommandRunner(client DockerInterface) ContainerCommandRun
> >  }
> >  
> >  func (p dockerPuller) Pull(image string) error {
> > -	image, tag := parseImageName(image)
> > +	_, tag := parsers.ParseRepositoryTag(image)
> I believe the issue is here, we removed the image output here (which was shadowing the image arg :( ) so we ask to pull Repository: "kubernetes/pause:0.1.0" instead of "kubernetes/pause" with tag "0.1.0"
> 
> —
> Reply to this email directly or view it on GitHub.
>